src/Data/List/Properties.agda

```diff
@@ -766,11 +770,20 @@ length-drop zero    xs       = refl
 length-drop (suc n) []       = refl
 length-drop (suc n) (x ∷ xs) = length-drop n xs
 
+------------------------------------------------------------------------
+-- take-drop
```

Comment:
No need for this separate section. It's fine to have them under `drop` as it comes after the `take` section.